MaterialDesignExtensions/Controllers/FileSystemController.cs

```diff
@@ -208,7 +208,15 @@ public List<SpecialDrive> Drives
                             label = label.Substring(0, label.Length - 1);
                         }
 
-                        string volumeLabel = driveInfo.IsReady ? driveInfo.VolumeLabel : null;
+                        string volumeLabel = null;
+                        try
+                        {
+                            volumeLabel = driveInfo.IsReady ? driveInfo.VolumeLabel : null;
+                        }
+                        catch (IOException exception) when (exception.Message.StartsWith("An unexpected network error occurred"))
```

Comment:
@dtwk2 I'm afraid that your condition might not work for any Windows installation. For instance, I get German exception messages, because I'm using German locales. There might be also other exceptions than an `IOException` (see [DriveInfo.VolumeLabel documentation](https://docs.microsoft.com/en-us/dotnet/api/system.io.driveinfo.volumelabel?view=netframework-4.8#System_IO_DriveInfo_VolumeLabel)). At the moment, I would prefer to catch any kind of exception regardless of the cause.